manifest.json

```diff
@@ -21,7 +21,7 @@
     "requirements": {
         "yunohost": ">= 3.6.0"
```

Comment:
```suggestion
        "yunohost": ">= 3.8.0"
```

Reply:
I used $nodejs_path instead ;).
But that does not solve the initial issue I hoped it would, so no real need for this MR actually.

Reply:
Ugh sorry I misread the comment too fast /o\